packages/kbn-apm-synthtrace/src/scenarios/many_services.ts

```diff
@@ -14,8 +14,8 @@ import { withClient } from '../lib/utils/with_client';
 
 const ENVIRONMENT = getSynthtraceEnvironment(__filename);
 
-const scenario: Scenario<ApmFields> = async ({ logger }) => {
-  const numServices = 500;
+const scenario: Scenario<ApmFields> = async ({ logger, scenarioOpts }) => {
+  const { services: numServices = 500 } = scenarioOpts ?? {};
```

Comment:
```suggestion
const scenario: Scenario<ApmFields> = async ({ logger, scenarioOpts = { services: 500 } }) => {
  const numServices = scenarioOpts.services;```

Reply:
Another question, why don't you call it `numServices` instead of `services`?

Reply:
regarding the 1st comment, its just the same, hahah. i have no opinion. I will accept the suggestion.
2nd comment - You mean the CLI parameter, it's actually called the same in some other scenario files as well. That's why didn't wanted to touch too many things